modules/installation-and-upgrade/pages/container-deployment/snippet-step-deploy-podman-certs.adoc

```diff
@@ -34,3 +34,11 @@ mgradm install podman <FQDN> \
 For more information, see [command]``mgradm install podman --help``.
 
 --
+
++
+
+[NOTE]
+====
+If you use VM images as a migration target, here as the last step, execute the command [command]``mgradm migrate`` instead of [command]``mgradm install``.
+====
+
```

Comment:
Looking at the render, I would move this Note above.

Change the snippet:

```
. Deploy the {productname}.
+
> Here add the note <
+
Execute one of the following commands, depending on the SSL certificate variant (self-signed or third-party).
  Replace `<FQDN>` with your fully qualified domain name of the {productname} Server:
...
```

Reply:
I hope I got it right.  And I reformatted the preceding step to align notes better.